kernel/kernel-default/src/main/java/io/delta/kernel/parquet/ParquetConverters.java

```diff
@@ -163,12 +159,12 @@ public abstract static class BasePrimitiveColumnConverter
         protected int currentRowIndex;
         protected boolean[] nullability;
```

Comment:
I was thinking resetWorkingState() would override and reset these, and is called in implementations of getDataColumnVector. Not a big deal though just unifies code